pulsar-client-cpp/docker/build-wheel-file-within-docker.sh

```diff
@@ -18,19 +18,22 @@
 # under the License.
 #
 
-set -e
+set -e -x
 
 cd /pulsar/pulsar-client-cpp
 
-find . -name CMakeCache.txt | xargs rm
-find . -name CMakeFiles | xargs rm -rf
+find . -name CMakeCache.txt | xargs -r rm
+find . -name CMakeFiles | xargs -r rm -rf
 
 cmake . -DPYTHON_INCLUDE_DIR=/opt/python/$PYTHON_SPEC/include/python$PYTHON_VERSION \
         -DPYTHON_LIBRARY=/opt/python/$PYTHON_SPEC/lib \
-        -DLINK_STATIC=ON
+        -DLINK_STATIC=ON \
+        -DBUILD_TESTS=OFF
 
 make clean
-make _pulsar
+make _pulsar -j3
 
 cd python
 python setup.py bdist_wheel
+
+auditwheel repair dist/pulsar_client-*-$PYTHON_SPEC-linux_x86_64.whl
```

Comment:
a comment on why auditwheel is needed would be useful.

Reply:
Sure, will add